### ISSUE TITLE
`max-width` on `<ContactFooter />` and `<Footer />`'s inner-container

### ISSUE THREAD

**Issue by cobraz** [bjerkio/website#126]
![image](https://user-images.githubusercontent.com/3726815/112982975-66589700-915d-11eb-8246-2b7cdb3d96e2.png)

The `<ContactFooter />` should scale to a maximum width (e.g. `1920px` like other elements). On the illustration/screenshot above, there are two a box (with red border). This box shows the container, of which you should find the title («Si hei») and other content. The background should be 100% of the window.

In other words:

https://github.com/bjerkio/website/blob/f8f1f25402d99c37ed96899f37a46d1529bd0b8d/web/src/components/layouts/components/contact-footer/contact-footer.tsx#L11-L15

Somewhere there 👆 it should be a `maxWidth: '100%'` (and probably some way to place it in center)

/cc @AnneMatilde @dkostenko-bjerk 

re #123
